torch/_torch_docs.py

```diff
@@ -780,14 +780,14 @@
            r"""
 clamp(input, min, max, out=None) -> Tensor
 
-Clamp all elements in :attr:`input` into the range `[min, max]` and return
+Clamp all elements in :attr:`input` into the range `[` :attr:`min`, :attr:`max` `]` and return
```

Comment:
Does this render correctly? If you want to outputs the brackets verbose then you need double backticks. Also does attr work for function arguments?

Reply:
Yes, they do.

![image](https://user-images.githubusercontent.com/23639302/37283200-1c62584c-261d-11e8-8b99-a3f97c68f274.png)

Please have look, and then I can make changes if required.

Reply:
Single backtick means you apply the restructured text "default role", which is most commonly italics.

Reply:
Would you like me to make changes?